### PR TITLE
Issue 1028 UI improvement 8

### LINKED ISSUE #1027: UI improvement #7

### LINKED ISSUE #1028: UI improvement #8

### PR DESCRIPTION
**Pull request by Ekaanth**
fix https://github.com/litentry/litentry-app/issues/1028 and https://github.com/litentry/litentry-app/issues/1027

### CODE REVIEW

**Comment by silva-fj** [comment]
What about point 1 and 2 of https://github.com/litentry/litentry-app/issues/1027 ?

**Comment by Ekaanth** [comment]
> What about point 1 and 2 of #1027 ?

The UI looks distorted when placed like this:



since we are using `StatInfoBlock` component and placing the text side by side would have a different UI.

**Comment by Ekaanth** [comment]
UPDATE:

File: src/ui/screens/Parachains/OverviewScreen.tsx
Comment (silva-fj) [inline src/ui/screens/Parachains/OverviewScreen.tsx]: Does this fit in a small screen? How does it look?

Comment (Ekaanth) [reply src/ui/screens/Parachains/OverviewScreen.tsx]: It looks like this on 4'5 inch screen: 


We need to reorder the UI in a better way to handle it.

Comment (silva-fj) [reply src/ui/screens/Parachains/OverviewScreen.tsx]: You can use the current approach for now (display and take space vertically), but properly separate items (label and value)

Comment (Ekaanth) [reply src/ui/screens/Parachains/OverviewScreen.tsx]: With the latest update this is how the UI looks:

Comment (silva-fj) [reply src/ui/screens/Parachains/OverviewScreen.tsx]: It does not look aligned, let's have a chat to revisit this

**Comment by Ekaanth** [comment]
with adding padder: The looks better but still not that great. 
UPDATE:

**Comment by ishrak-imam** [comment]
@Ekaanth @franjs I am taking a look into this. Will add commits here if I make any changes.

**Comment by Ekaanth** [comment]
> @Ekaanth @franjs I am taking a look into this. Will add commits here if I make any changes.

@ishrak-imam I had a mock UI what do you and @franjs think of this UI?

**Comment by ishrak-imam** [comment]
> 

Where did you get the mock from?

**Comment by ishrak-imam** [comment]
> > @Ekaanth @franjs I am taking a look into this. Will add commits here if I make any changes.
> 
> @ishrak-imam I had a mock UI what do you and @franjs think of this UI?
> 
> 

Where did you get the mock from?

**Comment by Ekaanth** [comment]
@franjs had asked me to play along with it to fit the best so I just tried something of my own.

**Comment by ishrak-imam** [comment]
Came up with this layout for the Parachains overview screen. @franjs @Ekaanth let me know what you think.

Also the progress chart was buggy. The percentage was not being displayed properly.

**Comment by Ekaanth** [comment]
@ishrak-imam The mockup looks amazing but where would you display `remainder`?

**Comment by ishrak-imam** [comment]
> @ishrak-imam The mockup looks amazing but where would you display `remainder`?

It’s inside the progress chart display.

**Comment by Ekaanth** [comment]
> > @ishrak-imam The mockup looks amazing, but where would you display `remainder`?
> 
> It’s inside the progress chart display.

IMO it doesn't provide any information like not sure if those maydays are left or completed.🤔

**Comment by ishrak-imam** [comment]
> > > @ishrak-imam The mockup looks amazing, but where would you display `remainder`?
> > 
> > 
> > It’s inside the progress chart display.
> 
> IMO it doesn't provide any information like not sure if those maydays are left or completed.🤔

But it is actually related to the the progress percentage.

**Comment by Ekaanth** [comment]
This UI does look good. 👍🏾

File: src/ui/components/ProgressWidget.tsx
Comment (silva-fj) [inline src/ui/components/ProgressWidget.tsx]:
```suggestion
        <Caption style={[styles.chartText, chartTextStyle]}>{detail}</Caption>
```

**Comment by ishrak-imam** [comment]
@franjs took the remaining time out the chart and moved the parathreads count into the parathread screen.
Can be merged once you approve.

**Review by silva-fj — APPROVED** [review]

**Review by ishrak-imam — APPROVED** [review]

**Pull request merged**
